### PR TITLE
drivers/sdcard: Add CRC values for all self.cmd() calls

### PR DESCRIPTION
**Pull request by dweick**
Some SD cards, such as the Samsung 64GB EVO SDXC, check the CRC on all self.cmd() calls in sdcard.py driver and return a CRC error result if they are set to 0x00 resulting in the code failing.  Adding the CRC values does not affect cards that don't use them so there is no harm in having them.

### CODE REVIEW

File: drivers/sdcard/sdcard.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def init_card(self):
 
         # get the number of sectors
         # CMD9: response R2 (R1 byte + 16-byte block read)
-        if self.cmd(9, 0, 0, 0, False) != 0:
+        if self.cmd(9, 0, 0, 0xAF, False) != 0:
```
Comment (dweick) [inline drivers/sdcard/sdcard.py]: Should be:
if self.cmd(9, 0, 0xAF, 0, False) != 0:

**Comment by dpgeorge** [comment]
Thanks for the contribution, this looks like a good improvement.  Ideally the CRC would be calculated on-the-fly but that would slow the code down a lot.  So what is done here is good.

**Comment by dpgeorge** [comment]
I tested these changes and it worked with a PYBD and a 4GB SD card.

File: drivers/sdcard/sdcard.py [hunk 2]
```diff
@@ -231,15 +231,15 @@ def readblocks(self, block_num, buf):
         assert nblocks and not len(buf) % 512, "Buffer length is invalid"
         if nblocks == 1:
             # CMD17: set read address for single block
-            if self.cmd(17, block_num * self.cdv, 0, release=False) != 0:
+            if self.cmd(17, block_num * self.cdv, 0x3B, release=False) != 0:
```
Comment (dpgeorge) [inline drivers/sdcard/sdcard.py]: Won't the CRC here depend on the value of `block_num * self.cdv`?  Same with the 3 other lines below.

**Comment by tve** [comment]
can someone fix the PR title? E.g. prefix with `drivers/sdcard` or something like that?

**Comment by dpgeorge** [comment]
Closing due to inactivity.  Feel free to reopen with responses to the above review comments.

**Pull request closed without merging**